File: frontend/lib/models/api/brand.dart
class Brand {
  final int brandId;
  final String brandName;
  final String abbreviation;
  final int latestSequence;

  Brand({
    required this.brandId,
    required this.brandName,
    required this.abbreviation,
    required this.latestSequence});

  factory Brand.fromJson(Map<String, dynamic> json) {
    return Brand(
      brandId: json['brandId'],
      brandName: json['brandName'],
      abbreviation: json['abbreviation'],
      latestSequence: json['latestSequence']
    );
  }

   Map<String, dynamic> toJson() => {
    'brandId': brandId,
    'brandName': brandName,
    'abbreviation': abbreviation,
    'latestSequence': latestSequence
  };
}

File: lis/src/main/java/com/biotech/lis/Entity/Brand.java
package com.biotech.lis.Entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "`brand`")
public class Brand {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer brandId;
    @Column(unique = true)
    private String brandName;
    private String abbreviation;
    private Integer latestSequence;
}
